src/sync.rs

```diff
@@ -414,6 +440,8 @@ mod tests {
     static DEFAULT_SYNC_OPTIONS: SyncOptions = SyncOptions {
         sync_reblogs: true,
         sync_retweets: true,
+        sync_hashtag_twitter: std::option::Option::None,
```

Comment:
can just be `None` without the fully qualified name, as we use otherwise in the code base. Please do that for all Option/Some/None.